content/800-guides/1300-using-prisma-in-docker.mdx

```diff
@@ -0,0 +1,510 @@
+---
+title: "Using Prisma in Docker"
```

Comment:
can you adjust the title so that it lines up with the "How to ..." wording and be consistent with the other titles (as discussed [here](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1736864954913649)).

I think you may also need to do this for some other guides you published recently actually.

<img width="767" alt="image" src="https://github.com/user-attachments/assets/5ef35886-5c29-42e3-9c75-47d38a4baa67" />